cmd/corectl/main.go

```diff
@@ -130,36 +142,34 @@ func configGenerator(db *sql.DB, args []string) {
 				fatalln("error:", "bad ed25519 public key length")
 			}
 			url := args[i+1]
-			signers = append(signers, config.BlockSigner{
+			signers = append(signers, &config.BlockSigner{
 				Pubkey: pubkey,
-				URL:    url,
+				Url:    url,
```

Comment:
Not that it's any consolation, but "Go style" makes an exception for this case:

> Code generated by the protocol buffer compiler is exempt from this rule. Human-written code is held to a higher standard than machine-written code.

https://github.com/golang/go/wiki/CodeReviewComments#initialisms